Makefile

```diff
@@ -0,0 +1,14 @@
+tools:
+	bash ./scripts/tools.sh
+
+lint:
+	bash ./scripts/lint.sh
+
+fix:
+	bash ./scripts/fix.sh
+
+tests:
+	gotestsum --no-color=false --format testname -- -timeout 600s -p 8 -parallel 8 -v -race -coverprofile=/tmp/profile.out ./pkg/...
```

Comment:
Can you give me an example? I have no clue how to do this in a Makefile.

Reply:
```
TEST_VERBOSE="${TEST_VERBOSE:=false}"
TEST_LOG_FORMAT="${TEST_LOG_FORMAT:=testname}"
TEST_PACKAGE="${TEST_PACKAGE:=./...}"
TEST_ARGS="${TEST_ARGS:=}"
if [[ $TEST_VERBOSE == "true" ]]; then
  TEST_ARGS="$TEST_ARGS -v"
fi

# Run tests, sequentially because the API is shared resource
echo "Running tests ..."
cmd="gotestsum --no-color=false --format \"$TEST_LOG_FORMAT\" -- -timeout 600s -p 8 -parallel 8 $TEST_ARGS "$TEST_PACKAGE" $@"
```

Reply:
try to remove `$@` and place there `...` or `./pkg/...` I think you miss what you want to test.

Reply:
Nope, still the same error.

Reply:
https://github.com/keboola/go-cloud-encrypt/pull/3